packages/apps/docs/src/pages/pact/index.tsx

```diff
@@ -47,7 +47,7 @@ const Home: FC<IProps> = ({ popularPages, blogPosts }) => {
 
         <Grid.Root gap="$lg" columns={{ sm: 1, lg: 2 }}>
           <Grid.Item>
-            <Card>
+            <Card fullWidth>
```

Comment:
I believe this should be `<Card expanded>`. We will update the notifications in the future to hopefully make this more clear

Reply:
it seems that expand does not exist. its called fullWidth as far as I can see

Reply:
Think so too, I see `fullWidth` but no `expanded`.